src/index.ts

```diff
@@ -1,3 +1,4 @@
-export { OpenWebProvider } from './components/OpenWebProvider';
-export { Conversation } from './components/Conversation';
-export { MessagesCount } from './components/MessagesCount';
+export { OpenWebProvider } from "./components/OpenWebProvider";
+export { Conversation } from "./components/Conversation";
+export { MessagesCount } from "./components/MessagesCount";
+export { startTTH, logout } from "./utils/TwoTokenHandshake";
```

Comment:
Are we ok with exporting these as plain javascript functions instead of React hook? I think that was the request at the beginning.

Reply:
I started with writing a hook. but there is no logical point with creating a hook here.

From my understanding, the main issue is to not call window.SPOTIM.<something> explicitly.

@dortzur , @shovalk WDYT?

Reply:
@shaybs247 Yeah you are correct :)

Reply:
💪🏻